### PR TITLE
Add Linter and CI

### PR DESCRIPTION
**Pull request by abhsag24**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 pics/*
-*.pyc
\ No newline at end of file
+*.pyc
+*.orig
```
Comment (nkprince007) [inline .gitignore]: mind the newline at the end of the file, please

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "3.4"
+# command to install dependencie
+sudo: true
+before_script:
+  - pip install coala
+script:
+  - coala-ci
```
Comment (nkprince007) [inline .travis.yml]: same newline here bro..

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -12,6 +12,11 @@ shortlog: commit message
 commit body
 Fixes <issue number>
 ```
+before you push remember to run
+```
+coala
+```
+and commit all changes suggested
```
Comment (nkprince007) [inline CONTRIBUTING.md]: add how to install `coala` too bro

File: .gitignore (hunk 1 above)
Comment (abhsag24) [inline .gitignore]: added

File: README.md [hunk 4]
```diff
@@ -38,4 +38,40 @@ https://gitter.im/NIT-dgp/General
...
+```
+shortlog: commit message
+
+commit body
+Fixes <issue number>
+```
+
+UIP uses coala as a linter, to install use:
+```
+sudp pip install coala
```
Comment (nkprince007) [inline README.md]: sudo is mistyped, bro

Comment (abhsag24) [inline README.md]: aahh nice

**Comment by nkprince007** [comment]
I think we could wrap this up now.. and merge the PR.

**Comment by nkprince007** [comment]
But it'd be better if we squash the commits, right?

**Comment by abhsag24** [comment]
alright so big change coming! :P

**Comment by abhsag24** [comment]
@nkprince007 no need two squash those are actually two different logical changes. Its better we keep them seperate.

**Comment by nkprince007** [comment]
Ok.. bro.

**Pull request merged**